### PR TITLE
enhance: improve check health

### PR DESCRIPTION
**Pull request by jaime0815**
issue: #34264

### CODE REVIEW

File: internal/querycoordv2/utils/util.go [hunk 1]
```diff
@@ -99,17 +99,14 @@ func GetShardLeaders(m *meta.Meta, targetMgr *meta.TargetManager, dist *meta.Dis
 		err := merr.WrapErrCollectionNotFullyLoaded(collectionID)
 		msg := fmt.Sprintf("collection %v is not fully loaded", collectionID)
 		log.Warn(msg)
-		return nil, err
-	}
-
-	channels := targetMgr.GetDmChannelsByCollection(collectionID, meta.CurrentTarget)
```
Comment (bigsheeper) [inline internal/querycoordv2/utils/util.go:105]: Why was this check removed?

File: internal/querycoordv2/utils/util.go [hunk 2]
```diff
@@ -82,12 +82,12 @@ func CheckLeaderAvailable(nodeMgr *session.NodeManager, leader *meta.LeaderView,
 	return nil
 }
 
-func GetShardLeaders(m *meta.Meta, targetMgr *meta.TargetManager, dist *meta.DistributionManager, nodeMgr *session.NodeManager, collectionID int64) ([]*querypb.ShardLeadersList, error) {
+func CheckLoadStatus(m *meta.Meta, collectionID int64) error {
```
Comment (bigsheeper) [inline internal/querycoordv2/utils/util.go:85]: make this function private.

File: pkg/util/paramtable/quota_param.go [hunk 3]
```diff
@@ -1584,15 +1584,10 @@ specific conditions, such as memory of nodes to water marker), ` + "true" + ` me
 		Version:      "2.2.0",
 		DefaultValue: defaultMaxTtDelay,
 		Formatter: func(v string) string {
-			if !p.TtProtectionEnabled.GetAsBool() {
```
Comment (bigsheeper) [inline pkg/util/paramtable/quota_param.go:1587]: Why was this check removed?

Comment (jaime0815) [reply pkg/util/paramtable/quota_param.go:1587]: It is so wired to return math.MaxInt64 if TtProtectionEnabled is false, which should return a configured value.

**Pull request merged**